telegram/inline/inlinequeryresultvenue.py

```diff
@@ -41,8 +45,8 @@ class InlineQueryResultVenue(InlineQueryResult):
         address (:obj:`str`): Address of the venue.
         foursquare_id (:obj:`str`): Optional. Foursquare identifier of the venue if known.
         foursquare_type (:obj:`str`): Optional. Foursquare type of the venue, if known.
-            (For example, "arts_entertainment/default", "arts_entertainment/aquarium" or
-            "food/icecream".)
```

Comment:
hmm I like this actually. Why did you remove it, maintaining?

Reply:
Because as a rule of thumb, attribute docstrings should be shorter than the corresponding argument docstring. at least that's what I did up to now …